libfreerdp/core/transport.c

```diff
@@ -371,6 +371,37 @@ BOOL transport_accept_nla(rdpTransport* transport)
 	return TRUE;
 }
 
+void transport_bio_error_log(char *biofunc, rdpTransport* transport)
```

Comment:
@giox069 ```WLog_*``` functions also record where they were called from. This wrapper function negates this feature, maybe it would be a good idea to add ```__FUNCTION__, __LINE__ and __FILE__``` as arguments and use ```WLog_PrintMessage``` directly.